src/main/java/uk/gov/hmcts/ccd/DefaultCacheWarmUpService.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.ccd;
```

Comment:
can we move this into a `cache` package?

Reply:
Done.

Reply:
Do you want the `CachingConfiguration` to go to `cache` package too?

Reply:
yes would be a good move 👍

Reply:
Cool